src/com/lftechnology/java/training/krishna/employeemanagement/controllers/LoginController.java

```diff
@@ -0,0 +1,479 @@
+				"**   4. Quit OR Logout                            **" + "\n" +
+				"**   5. Search User                               **" + "\n" +
+				"**   6. Edit Information                          **" + "\n" +
+				"****************************************************" + "\n");
+
+			chooseNormalUserAction(scanner, id);
+		}
+
+	}
+
+	/**
+	 * This method is used to choose admin user action such as add employee,
+	 * delete user
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @param id
+	 *            {@link int}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void chooseAdminUserAction(Scanner scanner, int id) {
+
+		int choice = ValidationUtils.numberValidation(scanner);
+
+		switch (choice) {
+		case 1:
+			addEmployee(scanner);
+			break;
+		case 2:
+			deleteUser(scanner);
+			break;
+		case 3:
+			terminateUser(scanner);
+			break;
+		case 4:
+			authentiateUser(scanner);
+			break;
+		case 5:
+			searchEmployee(scanner);
+			break;
+		default:
+			System.out.println(ConstantUtils.DESIRED_ACTION + "\n");
+			break;
+		}
+		showMainMenu(scanner, id);
+	}
+
+	/**
+	 * This method is used to choose normal user action such as search employee
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @param id
+	 *            {@link int}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void chooseNormalUserAction(Scanner scanner, int id) {
+
+		int choice = ValidationUtils.numberValidation(scanner);
+
+		switch (choice) {
+		case 4:
+			authentiateUser(scanner);
+			break;
+		case 5:
+			searchEmployee(scanner);
+			break;
+		case 6:
+			editEmployee(scanner, id);
+			break;
+		default:
+			System.out.println(ConstantUtils.DESIRED_ACTION + "\n");
+			break;
+		}
+		showMainMenu(scanner, id);
+	}
+
+	/**
+	 * This method is used to add employee from user input
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void addEmployee(Scanner scanner) {
+
+		System.out.println("************ Collecting Employee Information ************\n");
+		System.out.println("Please enter the username::");
+		String username =
+			ValidationUtils.emptyValidation(scanner, ConstantUtils.USERNAME);
+		System.out.println("Please enter the password::");
+		String password =
+			ValidationUtils.emptyValidation(scanner, ConstantUtils.PASSWORD);
+		System.out.println("Please enter the full name::");
+		String fullname =
+			ValidationUtils.emptyValidation(scanner, ConstantUtils.FULLNAME);
+		System.out.println("Please enter the department::");
+		String department =
+			ValidationUtils.emptyValidation(scanner, ConstantUtils.DEPARTMENT);
+		System.out.println("Please enter the address::");
+		String address =
+			ValidationUtils.emptyValidation(scanner, ConstantUtils.ADDRESS);
+		System.out.println("Please choose the following role::\n");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)